manuscript/06.methods.md

```diff
@@ -1,3 +1,22 @@
 ## Methods
 
 All analysis was implemented in Python v3.8 and can be found at <https://github.com/meyer-lab/tfac-ccle>.
+
+### Data Import & Normalization
+We imported protein characterization data from the LINCS dataset, provided by Drs. Joe Gray, Laura Heiser, and James Korkola. The data came in three separate files, so they were concatenated together into one protein dataset.
+	
+For the gene expression dataset, we imported Oregon Health and Science University data which gave us a variety of biological markers’ data. For the genetic proponent, we used the RNA sequence data given by the OHSU data.
+
+Protein and gene expression data were not normalized after importation since it provided no significant difference in tensor factorization outcome.
+
+### GSEA 
+We tested for enrichment along the ARCHS4 gene list, which contains gene expression measurements from a breadth of human and mouse experiments: ARCHS4 was selected for our analysis since it is a comprehensive list of commonly tested RNA expression genes.5
```

Comment:
What are the numbers at the end of these sentences?

Reply:
That's on me, I left them there in the Google Doc to remind myself about the references, but forgot to remove them from the text. Just removed them from the manuscript!